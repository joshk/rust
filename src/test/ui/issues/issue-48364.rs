fn foo() -> bool {
    b"".eq_ignore_ascii_case(stringify!(foo))
    //~^ ERROR mismatched types
}

fn main() {}
