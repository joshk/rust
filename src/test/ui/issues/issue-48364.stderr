error[E0308]: mismatched types
  --> $DIR/issue-48364.rs:2:30
   |
LL |     b"".eq_ignore_ascii_case(stringify!(foo))
   |                              ^^^^^^^^^^^^^^^ expected slice, found str
   |
   = note: expected type `&[u8]`
              found type `&'static str`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
